mkdocs.yml

```diff
@@ -53,7 +53,8 @@ markdown_extensions:
 
 nav:
   - 'Overview': index.md
-  - '1.x API': 1.x/paparazzi/app.cash.paparazzi/index.html
+  - '1.x API': 1.x/paparazzi/app.cash.paparazzi/index.md
```

Comment:
making sure this was intentional, is the 1.x no longer using html?

Reply:
reverting